content/partials/general/push/_push_intro.textile

```diff
@@ -10,17 +10,17 @@ As shown above, Ably provides two models for delivering push notifications to de
 
 h3(#direct-publishing). Direct publishing
 
-Ably provides a REST API that allows native push notifications to be delivered directly to:
+Ably provides a REST API that allows push notifications to be delivered directly to:
 
 * Devices identified by their unique device ID
 * Devices identified by their assigned "@clientId@":/realtime/authentication#identified-clients
-* Devices identified by the recipient details of the native push transport such as their unique @registrationToken@ in the case of FCM, @deviceToken@ in the case of APNS, or @targetUrl@ and @encryptionKey@ in the case of a Web device (*experimental*). This means is particularly useful when migrating to Ably with existing push notification target devices.
+* Devices identified by the recipient details of the push transport such as their unique @registrationToken@ in the case of FCM, @deviceToken@ in the case of APNS, or @targetUrl@ and @encryptionKey@ in the case of a Web device (*experimental*). This means is particularly useful when migrating to Ably with existing push notification target devices.
```

Comment:
Can we change "this means is" (easy to misread intent):

Maybe: "This approach is particularly useful when you are migrating applications that already use push notifications for target devices, to Ably."

Reply:
I think this one is outside the scope of this PR. Will address in the content review of that page.